RegexFileSearcher/RegexFileSearcher/FileHandler.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace RegexFileSearcher
+{
+    class FileHandler
```

Comment:
```suggestion
    internal class FileHandler
```
Explicitly adding the `internal` accessibility modifier here makes it consistent with the other classes and also satisfies the  analyzers.